docs/development.md

```diff
@@ -142,9 +181,24 @@ Sometimes, Eclipse does not show these options immediately after you set up the
 
 New developers should run `All tests` and have all of them passing at least once on their local environments.
 
-When running the test cases, if a few cases fail (this can happen due to timing issues), run the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.
+When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.
```

Comment:
Looks like the two portions are largely identical. Consider making it like:
```markdown
### Test Configurations

Several configurations are provided by default:
...

Additionally, ... godmode ...

New developers should ... local environments.

### Running the test suite with Eclipse

Tests can be run using the configurations available under the green `Run` button on the Eclipse toolbar.

When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.

### Running the test suite with IntelliJ

Tests can be run using the configurations available under `Run → Play`.

When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab in IntelliJ until they pass.
```